example/lib/widgets/location_page.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:flutter/material.dart';
+
+import 'package:another_location_plugin/another_location_plugin.dart';
+import '../utils/strings.dart';
+import '../utils/dimensions.dart';
+import '../utils/styles.dart';
+import 'main_button.dart';
+import 'secondary_button.dart';
+
+class LocationPage extends StatefulWidget {
+  const LocationPage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  _LocationCardState createState() => _LocationCardState();
+}
+
+class _LocationCardState extends State<LocationPage> {
+  bool hasPermissions = false;
+  bool isInitiallized = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        image: DecorationImage(
+          image: AssetImage(
+            Strings.backgroundImageUrl,
+          ),
+          fit: BoxFit.cover,
+        ),
+      ),
+      child: Center(
+        child: SizedBox(
+          width: Dimensions.locationCardContentWidth,
+          height: Dimensions.locationCardContentHeight,
+          child: Padding(
+            padding: const EdgeInsets.all(
+              Dimensions.locationCardContentPadding,
+            ),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Text(
+                      Strings.locationCardTitle,
+                      style: TextStyle(
+                        fontSize: Styles.locationCardTitleFontSize,
+                        fontWeight: FontWeight.w500,
+                        color: Colors.grey.shade100,
+                        shadows: [
+                          Shadow(
+                            color: Colors.black,
+                            blurRadius: Styles.locationCardTextShadowBlur,
+                          ),
+                        ],
+                      ),
+                      textAlign: TextAlign.left,
+                    ),
+                  ],
+                ),
+                StreamBuilder(
+                  stream: AnotherLocationPlugin.locationStream,
+                  initialData: Strings.locationCardHint,
+                  builder: (context, snapshot) {
+                    return snapshot.hasData
+                        ? Row(
```

Comment:
This row is necessary? Remember that row, columns, etc with only one child lose sense

Reply:
Wrongly used for alignment.. Replaced by property!